File: src/main/java/com/halehan/demo/rest/model/Patient.java
package com.halehan.demo.rest.model;

import javax.persistence.*;


@Entity // This tells Hibernate to make a table out of this class
public class Patient {
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private Integer id;

    private Integer studyid;

    private String firstname;

    private String lastname;

    private String email;

    private String status;

    private String studyeye;

    public String getStudyeye() {
        return studyeye;
    }

    public void setStudyeye(String studyeye) {
        this.studyeye = studyeye;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getStudyid() {
        return studyid;
    }

    public void setStudyid(Integer studyid) {
        this.studyid = studyid;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: src/main/java/com/halehan/demo/rest/model/Patient.kt
package com.halehan.demo.rest.model

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity // This tells Hibernate to make a table out of this class
class Patient {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    var id: Int? = null
    var studyid: Int? = null
    var firstname: String? = null
    var lastname: String? = null
        private set
    var email: String? = null
    var status: String? = null

    fun setLastName(lastName: String?) {
        lastname = lastName
    }

}